index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Final projects</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <header>
+    <nav class="nav__bar" id="navbar">
+      <ul>
+        <li><a class="nav__links" href="#welcome-section">About</a></li>
+        <li><a class="nav__links" href="#projects">work</a></li>
+        <li><a class="nav__links" href="#cantact">cantact</a></li>
+      </ul>
+    </nav>
+  <section class="section__first" id="welcome-section">
+    <h1 class="section__h1">Hey I am Roya Jamal </h1>
+    <p class="section__p"> A web developer</p>
+  </section>
+  </header>
+  <section class="section__projects" id="projects">
+    <h2>These are some of my projects</h2>
+    <a id="profile-link" rel="noopener"  href="https://github.com/royajamal" target="_blank" class="project-tile"> visite my GitHub profile</a>
+</section>
```

Comment:
- Opening tag and closing tag of section is not aligned correctly, please make the alignment correct (add 2 spaces before closing tag)